src/unicode2latex.jl

```diff
@@ -2450,26 +2506,39 @@ const unicodedict = Dict{Char, String}(
     'Χ' => raw"\Chi",
     '⤮' => raw"\neovsearrow",
     '•' => raw"\bullet",
-    )
+    (latex_diacritics.('a':'z')...)...,
```

Comment:
Is this not the same thing as just 
```suggestion
    latex_diacritics.('a':'z')...,
```
?

Reply:
No, because you have to splat (unpack) all diacritics for a single char, and then do that for all chars in the range `'a':'z'`.